tests/check_framework/test_migrations.py

```diff
@@ -0,0 +1,61 @@
+from django.core import checks
+from django.core.checks.migrations import check_migration_operations
+from django.db import migrations
+from django.db.migrations.operations.base import Operation
+from django.test import TestCase
+
+
+class DeprecatedMigrationOperationTests(TestCase):
+    def test_default_operation(self):
+        class MyOperation(Operation):
+            system_check_deprecated_details = {}
+
+        my_operation = MyOperation()
+
+        class Migration(migrations.Migration):
+            operations = [
+                my_operation,
+            ]
```

Comment:
This can be single-lined.

Reply:
> This can be single-lined.

is there a lint rule for such a case?

Reply:
I'd _guess_ is that this was automatically performed by `black` because you had a trailing comma within the list (i.e. `[my_operation,]` rather than `[my_operation]`) which [black will treat as a rule to make the expression multi-line](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#trailing-commas)
> A pre-existing trailing comma informs Black to always explode contents of the current bracket pair into one item per line.